backends/imgui_impl_win32.cpp

```diff
@@ -272,7 +272,7 @@ static void ImGui_ImplWin32_UpdateGamepads()
 
 static BOOL CALLBACK ImGui_ImplWin32_UpdateMonitors_EnumFunc(HMONITOR monitor, HDC, LPRECT, LPARAM)
 {
-    MONITORINFO info = { 0 };
+    MONITORINFO info = { 0, {0,0,0,0}, {0,0,0,0}, 0 };
     info.cbSize = sizeof(MONITORINFO);
```

Comment:
What is the warning here? Would using = `{};` fixes it?

Reply:
Under Clang / VisualStudio2019 / Warning 4
>1>..\..\backends\imgui_impl_win32.cpp(275,28): warning : missing field 'rcMonitor' initializer [-Wmissing-field-initializers]

`={};` also Works. Will change it to that.